coffee_core/src/coffee.rs

```diff
@@ -358,6 +389,34 @@ impl PluginManager for CoffeeManager {
         );
         Err(err)
     }
+
+    async fn nurse(&mut self) -> Result<(), CoffeeError> {
+        let repos_path = PathBuf::from(format!("{}/repositories/", self.config.root_path));
+        for entry in fs::read_dir(repos_path)? {
+            let entry = entry?;
+            let entry_name = entry.file_name().to_string_lossy().to_string();
+            if !self.repos.contains_key(&entry_name) {
+                if entry.file_type()?.is_dir() {
+                    fs::remove_dir_all(entry.path())?;
+                    log::debug!("folder removed: {}", entry_name);
+                } else if entry.file_type()?.is_file() {
+                    fs::remove_file(entry.path())?;
+                    log::debug!("file removed: {}", entry_name);
+                }
```

Comment:
add a comment here that if there is a missing file inside the repository we considered the repository corrupted, it is not trivial for the people that will go to read the  code